script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_5_1.txt

```diff
@@ -30,7 +30,7 @@ The dizziness shakes my brain.
 Just like that, I fall to my knees in a sea of red blood.
 <Page8>
 "Hurk――ughh――"
-My gastric juice refluxes back up.
+My gastric juice refluxes back up.//to be rechecked

```

Comment:
My acid reflux triggers.

Reply:
Hm, that sorta makes it sound like a preexisting condition. My gastric juices well up in my throat?

Reply:
I'm find with just "Gastric juices well up in my throat"